sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -218,9 +218,12 @@ class StreamSuite extends StreamTest {
       }
     }
 
-    val df = spark.readStream.format(classOf[FakeDefaultSource].getName).load()
-    assertDF(df)
-    assertDF(df)
+    // TODO: fix file source V2 as well.
+    withSQLConf(SQLConf.USE_V1_SOURCE_READER_LIST.key -> "parquet") {
+      val df = spark.readStream.format(classOf[FakeDefaultSource].getName).load()
```

Comment:
how is this related to parquet?

Reply:
```
[info]   Decoded objects do not match expected objects:
[info]   expected: WrappedArray(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
[info]   actual:   WrappedArray(9, 0, 10, 1, 2, 8, 3, 6, 7, 5, 4)
[info]   assertnotnull(upcast(getcolumnbyordinal(0, LongType), LongType, - root class: "scala.Long"))
[info]   +- upcast(getcolumnbyordinal(0, LongType), LongType, - root class: "scala.Long")
[info]      +- getcolumnbyordinal(0, LongType) (QueryTest.scala:70)
```
We need to fix the read path of steaming output.